inference_segmentation.py

```diff
@@ -404,6 +404,10 @@ def main(params: Union[DictConfig, dict]) -> None:
                             tp_mem=temp_file,
                             debug=debug)
 
+        # applie the mask 
+        mask = aoi.mask
+        pred[mask == 0] = 0
```

Comment:
can we write:
`pred[aoi.mask == 0] = 0 `
here?

Reply:
oups hahahah thanks